packages/web3-validator/src/validation/numbers.ts

```diff
@@ -36,6 +36,10 @@ export const isUInt = (
 		return false;
 	}
 
+	if (typeof value === 'string' && value.length === 0) {
```

Comment:
Just remove line 35,36, and 37 above and can use:
```suggestion
	if ( (!['number', 'string', 'bigint'].includes(typeof value))  ||
	    (typeof value === 'string' && value.length === 0) ) {
```